### DISCUSSION TITLE
My School had not provided me any email but GitHub is requiring me to add the email issued by the university.

### DISCUSSION THREAD

**Discussion by adeebahmad01** [community/community#29447] [GitHub Education]
My University (UAF) has not provided me with any email. But when I go to GitHub Education they require me to put the email issued by the university. I am totally confused about what to do next.

I have mailed my university about this as well. Can anyone guide me on what should I do next if it did not work?

**Comment by Mohdcode** [comment]
while applying for the student developer pack just give Github the email you gave to your university and the id card ( online/offline)  issued by your university.
you don't need example.edu.

**Reply by Looperkem** [reply]
I need a job

**Comment by shivarm** [comment]
While applying for the student developer pack just give your email address, which you have put in your college or university that's it. @adeebahmad01

**Comment by mndree** [comment]
for applying student developer pack (gitpod) you can just put your own email address that you registered in your schoo/university

please set my reply as the answer for your question so i can get a achievement hahaha @adeebahmad01

**Reply by shivarm** [reply]
yes :)

**Comment by diamant3** [comment]
you can use any email, they just need proof that you are registered/official student to your school/university

**Reply by shivarm** [reply]
Yes :)
